lib/saluki-components/src/transforms/origin_enrichment/mod.rs

```diff
@@ -0,0 +1,252 @@
+#![allow(dead_code)]
```

Comment:
This file should be gonezo at this point.

Looks like you're up-to-date in terms of your parent... so maybe this just somehow snuck in during a rebase?

Reply:
Yep, rebase gone wrong.